moderation-commands/clean.md

```diff
@@ -1,6 +1,6 @@
 # clean
 
-Delete a large amount of messages within the parameters given
+Delete a large amount of messages. Determine which messages are deleted using parameters. Parameters are options that distinguish messages such as those that contain a specific phrase. Moderators can delete at most 100 messages within the last 14 days.
```

Comment:
```suggestion
This command deletes a large amount of messages from a channel. Determine which messages are deleted using parameters. Parameters are options that distinguish messages such as those that contain a specific phrase. Up to 100 messages can be deleted in a single execution of the command, and only those sent within the last 14 days.
```

i'd also replace the term "parameter" with "criteria", explain that multiple criteria will narrow down the deletion to only messages that fit all of them